201-cosmosdb-advanced-threat-protection-create-account/README.md

```diff
@@ -0,0 +1,25 @@
+# Create an Azure CosmosDB Account with Advanced Threat Protection enabled
+
```

Comment:
what about links to the documentation?
is it common in arm templates?

Reply:
in our template for storage we have a link. do we we have such link for cosmos to be used here?